### DISCUSSION TITLE
Problem running docker ./manage.sh script.

### DISCUSSION THREAD

**Discussion by PaulSchulz** [uxbox/uxbox#1372] [General]
System: Ubuntu 21.10
I am trying to follow the devenv instructions and get to the following error when I try to rum **./manage.sh run-devenv**

  unknown shorthand flag: 'p' in -p
  See 'docker --help'.
  <snip>

Looking at the script, the following line is causing the error..

   docker compose -p $DEVENV_PNAME ...

I have followed the installation instructions for docker, 
$ docker -v
Docker version 20.10.11, build dea9396

$ docker compose 
docker: 'compose' is not a docker command.

but docker-compose is installed.
What am I doing wrong?

**Comment by PaulSchulz** [comment]
This appears to be an issue with not having Compose V2 installed.

  https://docs.docker.com/compose/cli-command/#installing-compose-v2

**Reply by PaulSchulz** [reply]
The fix (TL;DR)
 $ mkdir -p ~/.docker/cli-plugins/
 $ curl -SL https://github.com/docker/compose/releases/download/v2.0.1/docker-compose-linux-x86_64 -o ~/.docker/cli-plugins/docker-compose

$  chmod +x ~/.docker/cli-plugins/docker-compose

Test with
$ docker compose version

**Comment by hirunatan** [comment]
It's true. The devenv now requires docker-compose version 2. I've just updated the help center https://help.penpot.app/technical-guide/developer/devenv/

Note that this is only for the developer environment. The normal docker image works with 1.xx https://help.penpot.app/technical-guide/getting-started/
